src/Illuminate/Mail/MailManager.php

```diff
@@ -281,7 +281,7 @@ protected function createMailgunTransport(array $config)
         }
 
         return $factory->create(new Dsn(
-            'mailgun+https',
+            'mailgun+'.($config['transport_type'] ?? 'https'),
```

Comment:
This should be `scheme` to match Symfony's terminology.

```suggestion
            'mailgun+'.($config['scheme'] ?? 'https'),
```